update mdl gem

aaronmallen opened the pull request.

aaronmallen committed:
update mdl gem

maxjacobson commented:
Hi @aaronmallen , thanks for this PR. Looks like we'll need to do some more additional changes to make this work. Perhaps the output line format changed...

I see in 0.5.0 that mdl added a JSON formatter. That is _awesome_ as it makes it possible for us to parse the output from the underlying tool as a structured format instead of a plaintext format.

Do you have bandwidth to explore making the tests green, either by taking on that refactor to use the JSON formatter, or by updating the line pattern?

maxjacobson commented:
FWIW, I spent a few mins and figured out what the issue was... going to go ahead and push up the quick fix. The better fix would be to use the JSON formatter, but we can defer that unless you're interested in taking it on.

maxjacobson committed:
Adjust for 0.5.0

0.5.0 also introduced a JSON formatter, which would make our lives
easier. Just deferring biting that off because it's nice out.

aaronmallen commented:
sorry, just got back from a business trip :+1: good job merging this in.
